deployments/docker/src/builder-bashrc

```diff
@@ -0,0 +1,3 @@
+alias ll='ls -lh --color=auto'
```

Comment:
Should not this file and the below ones, should have the header for License statement?